CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 - Dropped support for `rxjs@5`. Consumers may install `rxjs-compat@^6` to support older versions of `rxjs`. [#20](https://github.com/blackbaud/skyux-autonumeric/pull/20)
 
+# 3.0.5 (2020-04-23)
```

Comment:
The changelog entries should be listed in order by date.

Reply:
@Blackbaud-SteveBrush good call. I noticed that on the other libraries but missed going back and fixing this first one.